R/query_all_versions.R

```diff
@@ -0,0 +1,114 @@
+#' Get all Solr fields
+#'
+#' @description Simple helper function for 'query_all_versions' function to retrive all Solr fields
+#'
+#' @return (character) List of Solr fields
+#'
+#' Written by Irene in the reference guide
+
+get_solr_fields <- function(){
+    adc_solr <- httr::GET("https://arcticdata.io/metacat/d1/mn/v2/query/solr")
+    suppressMessages(suppressWarnings(adc_solr<- adc_solr %>%
+                                          stringr::str_extract_all("name>.*<") %>%
+                                          unlist() %>%
+                                          stringr::str_replace_all("name>|<", "")))
+    return(adc_solr)
+}
+
+
+#' Create data frame with Solr fields
+#'
+#' This is a helper function for 'query_all_versions' function. It simplifies solr queries
+#' and creates a one row data frame with the specified Solr fields as the columns.
+#'
+#' @author Sharis Ochs
+#'
+#' @param node (character) Specify the node where the object should be searched for.
+#' @param object_pid (character) PID for the object that you want to return information about.
+#' @param fields (list) List of fields that you want returned in the data frame. Default
```

Comment:
in your example it looks like the input is a character vector.  this should be (character)